_episodes/03-skimage-images.md

```diff
@@ -101,6 +101,54 @@ image = skimage.io.imread(fname="chair.jpg")
 ~~~
 {: .python}
 
+> ## Import Statements in Python
+>
+> In Python, the `import` statement is used to load additional functionality
+> into a program. This is necessary when we want our code to do something more
+> specialised, which cannot easily be achieved with the limited set of basic
+> tools and data structures available in the default Python environment.
+>
+> Additional functionality can be loaded as a single function or object,
+> a module defining several of these, or a library containing many modules.
+> You will encounter several different forms of `import` statement.
```

Comment:
Would it make sense to enumerate the different forms in this sentence in the same order as they appear in the example? I'm in two minds on this, as both the sentence and the code block seem natural to me as they are.

Reply:
I agree that it would nice if the order in the sentence matched that in the code block. However, I have tried reversing the order in the text and found it a bit clunky.  The alternative is to reverse the order in the code block but that means switching from increasing to decreasing complexity, which I think makes it harder to understand. On balance, I would prefer to leave this order as it is. Open to suggestions from anyone who can find an elegant solution, though!